Utils/azure-explorer-common/src/com/microsoft/azure/toolkit/lib/appservice/file/AppServiceFileService.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) Microsoft Corporation
+ *
+ * All rights reserved.
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ * documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+ * to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of
+ * the Software.
+ *
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+ * THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.microsoft.azure.toolkit.lib.appservice.file;
+
+import com.google.common.base.Joiner;
+import com.microsoft.azure.management.appservice.WebAppBase;
+import com.microsoft.azure.management.appservice.implementation.AppServiceManager;
+import com.microsoft.rest.RestClient;
+import lombok.SneakyThrows;
+
+import java.lang.reflect.Method;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+public class AppServiceFileService {
+
+    private final AppServiceFileClient client;
+    private final WebAppBase app;
+
+    private AppServiceFileService(final WebAppBase app, AppServiceFileClient client) {
+        this.app = app;
+        this.client = client;
+    }
+
+    public static AppServiceFileService forApp(final WebAppBase app) {
+        final AppServiceFileClient client = getClient(app);
+        return new AppServiceFileService(app, client);
+    }
+
+    public List<? extends AppServiceFile> getFilesInDirectory(String path) {
+        return this.client.getFilesInDirectory(path).toBlocking().first().stream()
+                          .filter(file -> !"text/xml".equals(file.getMime()) || !file.getName().contains("LogFiles-kudu-trace_pending.xml"))
```

Comment:
why we need to filter the kudu log files?

Reply:
same as vscode
![image](https://user-images.githubusercontent.com/69189193/97246568-a0cda900-1838-11eb-965e-72381763fa57.png)

Reply:
add a comment to the original code